### PR TITLE
NAS-110029 / 21.04 / add system hash to public api

### PR DESCRIPTION
**Pull request by yocalebo**

### CODE REVIEW

File: src/middlewared/middlewared/plugins/system_/hostid.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+class SystemHostIdService(Service):
+    hostid = None
+
+    class Config:
+        namespace = 'system.host'
+
+    @accepts()
+    def id(self):
+        if self.hostid is None:
```
Comment (sonicaj) [inline src/middlewared/middlewared/plugins/system_/hostid.py:14]: Let's please add a docstring.

**Review by sonicaj — COMMENTED** [review]
Would we be extending `system.host` service ? If not, how about just having it under `system` namespace ?

File: src/middlewared/middlewared/plugins/system_/hostid.py (hunk 1 above)
Comment (yocalebo) [reply src/middlewared/middlewared/plugins/system_/hostid.py:20]: sure, no problem

**Review by sonicaj — APPROVED** [review]
Thanks!

**Pull request merged**